lib/ladybug/energy_model/extension.rb

```diff
@@ -83,7 +90,7 @@ def schema
         @@schema
       end
 
-    # check if the schema is valid
+    # check if the model schema is valid
       def schema_valid?
         if Gem.loaded_specs.has_key?("json-schema")
           require 'json-schema'
```

Comment:
@chriswmackey should there be an `else` statement here as well?

Reply:
I think if this function returned something, we would need the `else` statement.  But, because it's just running the OpenAPI schema through the JSON validator only when the `json-schema` is present (and not doing anything when it's not there), it should be fine.